response_operations_ui/views/respondents.py

```diff
@@ -85,7 +85,8 @@ def respondent_search():
                            respondents=filtered_respondents,
                            respondent_count=total_respondents_available,
                            first_index=1 + offset,
-                           last_index=results_per_page + offset,
+                           last_index=(results_per_page + offset) if total_respondents_available >= results_per_page
```

Comment:
This is a bit long, can this be moved to a separate variable above for clarity?

Reply:
changed